3459/0/triangle.rb

```diff
@@ -0,0 +1,42 @@
+# Triangle Project Code.
+
+# :reek:all
+# Triangle analyzes the lengths of the sides of a triangle
+# (represented by a, b and c) and returns the type of triangle.
+#
+# It returns:
+#   :equilateral  if all sides are equal
+#   :isosceles    if exactly 2 sides are equal
+#   :scalene      if no sides are equal
+#
+# The tests for this method can be found in
+#   about_triangle_project.rb
+# and
+#   about_triangle_project_2.rb
+#
+def triangle(side_a, side_b, side_c)
+  if check_statement(side_a, side_b, side_c)
+    type_of_triangle = process_legs(side_a, side_b, side_c)
+    return type_of_triangle.nil? ? :scalene : type_of_triangle
```

Comment:
you can solve it easier no need to check on nil you can revert condition and get it nicer :)